core/src/main/java/io/camunda/connector/impl/ReflectionHelper.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.connector.impl;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import java.util.Arrays;
+
+/** Utility class for reflection helper methods, like finding getters and setter for fields. */
+public class ReflectionHelper {
```

Comment:
Consider making it final

```suggestion
public final class ReflectionHelper {
```

Reply:
Also, this class is quite complex. Could we add some unit tests for it?

Reply:
It's actually tested quite extensively in the [OutboundConnectorContextTest](https://github.com/camunda/connector-sdk/pull/83/files#diff-263b4930ecc0ffdb06fbacf3f15dd947b3beddd989030662f97b9bfb755af1c0)

Reply:
Thanks for confirming.